export/serializers/csv_serializer.py

```diff
@@ -21,14 +21,21 @@
 import codecs
 import csv
 import re
-import StringIO
+import six
+try:
+    import StringIO
+except ImportError:
+    import io as StringIO
```

Comment:
I do not believe so, since the use is in the form `StringIO.StringIO(stream_or_string)`